### ISSUE TITLE
Invalid syntax after pip install

### ISSUE THREAD

**Issue by jpmens** [jpmens/mqttwarn#385]
```
$ python3 -mvenv v3
$ source v3/bin/activate
$ pip install mqttwarn
$ mqttwarn
Traceback (most recent call last):
  File "/private/tmp/m1/v3/bin/mqttwarn", line 11, in <module>
    load_entry_point('mqttwarn==0.11.3', 'console_scripts', 'mqttwarn')()
  File "/private/tmp/m1/v3/lib/python3.7/site-packages/pkg_resources/__init__.py", line 487, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/private/tmp/m1/v3/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2728, in load_entry_point
    return ep.load()
  File "/private/tmp/m1/v3/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2346, in load
    return self.resolve()
  File "/private/tmp/m1/v3/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2352, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/private/tmp/m1/v3/lib/python3.7/site-packages/mqttwarn/commands.py", line 12, in <module>
    from mqttwarn.configuration import load_configuration
  File "/private/tmp/m1/v3/lib/python3.7/site-packages/mqttwarn/configuration.py", line 123
    except Exception, e:
                    ^
SyntaxError: invalid syntax
```

**Comment by jpmens** [comment]
Replacing the comma by `as`, then results in

```
Traceback (most recent call last):
  File "/private/tmp/m1/v3/bin/mqttwarn", line 11, in <module>
    load_entry_point('mqttwarn==0.11.3', 'console_scripts', 'mqttwarn')()
  File "/private/tmp/m1/v3/lib/python3.7/site-packages/pkg_resources/__init__.py", line 487, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/private/tmp/m1/v3/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2728, in load_entry_point
    return ep.load()
  File "/private/tmp/m1/v3/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2346, in load
    return self.resolve()
  File "/private/tmp/m1/v3/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2352, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/private/tmp/m1/v3/lib/python3.7/site-packages/mqttwarn/commands.py", line 12, in <module>
    from mqttwarn.configuration import load_configuration
  File "/private/tmp/m1/v3/lib/python3.7/site-packages/mqttwarn/configuration.py", line 8, in <module>
    from ConfigParser import RawConfigParser, NoOptionError
ModuleNotFoundError: No module named 'ConfigParser'
```

**Comment by jpmens** [comment]
(apologies that I cannot be more constructive at the moment!)

**Comment by amotl** [comment]
Dear @jpmens,

Python 3 support has not landed yet, this is still ongoing within #336. Sorry that I wasn't clear on that.

However, installing `mqttwarn` through `pip` within a Python 2 virtualenv might make you happy already:
```
virtualenv --python=python2 .venv2
source .venv2/bin/activate
pip install mqttwarn
```

With kind regards,
Andreas.

**Comment by jpmens** [comment]
Thank you.
